### ISSUE TITLE
schedule.today in the FAQ's doesn't work for 1 off jobs

### ISSUE THREAD

**Issue by Trever123** [dbader/schedule#49]
The code snippet in the FAQ's for scheduling a job just once does not actually seem to work. I have just tried the following and got the error that the attribute today does not exist (i also checked the code and can't see today as part of scheduler or job class):

```
>>> import schedule
>>> def job():
...     print "its time"
...     return schedule.CancelJob
... 
>>> schedule.today.at('22:30').do(job)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'today'
```

Then to double check that scheduler is working for me i did the following:

```
>>> schedule.every().day.at('22:30').do(job)
Every 1 day at 22:30:00 do job() (last run: [never], next run: 2015-07-13 22:30:00)
```

I am wanting to schedule 1 off jobs for the future (as well as every day jobs) and wondering how this could be achieved.

Thanks,
James

**Comment by dbader** [comment]
Thanks @Trever123. You're right, that snippet is broken. Not sure why I ended up with a `today` there :) The line should read `schedule.every().day.at('22:30').do(job)` like you said. I'll fix the FAQ later. Thanks again!

**Comment by dbader** [comment]
Fixed in #50.
